tests/test_cases/coreutils_suite/popen_t.sh

```diff
@@ -0,0 +1 @@
+tempfolder/popen
```

Comment:
The way the popen test is handled is a little bit janky, but overall this is good.